NAMESPACE

```diff
@@ -44,9 +44,12 @@ export(bitly_update_group_pref)
 export(bitly_update_user)
 export(bitly_user_info)
 export(bitly_user_metrics_referring_domains)
+export(clipExpanderAddin)
+export(clipShortenerAddin)
 export(is_bitly_user_premium_holder)
 export(isgd_LinksExpand)
```

Comment:
I am not expert on Rstudio addins....please check it if ok (I am fine with both - just want to be on the safe side)

Reply:
Yep, must have slipped during last build - correct to include them in `NAMESPACE` ✅.